### PR TITLE
Deprecation fixes and LayoutInflater.from

### PR DESCRIPTION
**Pull request by TacoTheDank**
From https://github.com/cgeo/cgeo/pull/9531

### CODE REVIEW

**Comment by bekuno** [comment]
ok to test

File: main/src/cgeo/geocaching/CacheListActivity.java [hunk 1]
```diff
@@ -1536,9 +1536,9 @@ public void onClick(final View arg0) {
             showProgress(true);
             showFooterLoadingCaches();
 
-            getSupportLoaderManager().restartLoader(CacheListLoaderType.NEXT_PAGE.getLoaderId(), null, CacheListActivity.this);
+            LoaderManager.getInstance(CacheListActivity.this).restartLoader(CacheListLoaderType.NEXT_PAGE.getLoaderId(), null, CacheListActivity.this);
```
Comment (bekuno) [inline main/src/cgeo/geocaching/CacheListActivity.java:1538]: Why not use `LoaderManager.getInstance(this).` as with the other changes?

File: main/src/cgeo/geocaching/CacheListActivity.java [hunk 2]
```diff
@@ -1536,9 +1536,9 @@ public void onClick(final View arg0) {
...
             // the loader for subsequent pages takes over - therefore the initial loader needs to be destroyed
-            getSupportLoaderManager().destroyLoader(type.getLoaderId());
+            LoaderManager.getInstance(CacheListActivity.this).destroyLoader(type.getLoaderId());
```
Comment (bekuno) [inline main/src/cgeo/geocaching/CacheListActivity.java:1540]: Why not use `LoaderManager.getInstance(this).` as with the other changes?

**Review by bekuno — CHANGES_REQUESTED** [review]

File: main/src/cgeo/geocaching/utils/ClipboardUtils.java [hunk 3]
```diff
@@ -25,22 +26,23 @@ private ClipboardUtils() {
...
      * get clipboard content
      *
      */
-    @SuppressWarnings("deprecation")
     @Nullable
     public static String getText() {
         // fully qualified name used here to avoid buggy deprecation warning (of javac) on the import statement
-        final android.text.ClipboardManager clipboard = (android.text.ClipboardManager) CgeoApplication.getInstance().getSystemService(Context.CLIPBOARD_SERVICE);
+        final ClipboardManager clipboard = (ClipboardManager) CgeoApplication.getInstance().getSystemService(Context.CLIPBOARD_SERVICE);
         final CharSequence text = clipboard.getText();
```
Comment (bekuno) [inline main/src/cgeo/geocaching/utils/ClipboardUtils.java:46]: `clipboard.getText();` is also deprecated. Is there a chance to find a solution as fefore for `setText()`?

**Comment by bekuno** [comment]
Additional suggestion:
The `Intent.FLAG_ACTIVITY_CLEAR_WHEN_TASK_RESET` is also used in `ProcessUtils` and should be replaced

File: main/src/cgeo/geocaching/utils/ClipboardUtils.java (hunk 3 above)
Comment (TacoTheDank) [reply main/src/cgeo/geocaching/utils/ClipboardUtils.java:46]: Perhaps, though I'm not certain about how to go about it, so I'm leaving it as is so I don't break anything by accident.

**Comment by TacoTheDank** [comment]
Fixing conflicts, don't merge yet.

**Comment by moving-bits** [comment]
> Fixing conflicts, don't merge yet.

Thanks for fixing. To make the "don't merge yet" more visible, you can either set the "WIP / Do not merge" label or convert your PR to a draft.

File: main/src/cgeo/geocaching/CacheListActivity.java (hunk 1 above)
Comment (TacoTheDank) [reply main/src/cgeo/geocaching/CacheListActivity.java:1538]: The instance of the type variable has to exist, so `this` has to be explicitly referred to as the activity.

File: main/src/cgeo/geocaching/CacheListActivity.java (hunk 2 above)
Comment (TacoTheDank) [reply main/src/cgeo/geocaching/CacheListActivity.java:1540]: See above.

**Comment by TacoTheDank** [comment]
@bekuno All good now.

**Comment by bekuno** [comment]
LGTM now

**Comment by bekuno** [comment]
retest this please

**Review by bekuno — APPROVED** [review]

**Pull request merged**